ui/plugins/eu.esdihumboldt.hale.ui.common/src/eu/esdihumboldt/hale/ui/common/crs/AdvCrsAuthorityFactory.java

```diff
@@ -0,0 +1,363 @@
+/*
+ * Copyright (c) 2018 wetransform GmbH
+ * 
+ * All rights reserved. This program and the accompanying materials are made
+ * available under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ * 
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this distribution. If not, see <http://www.gnu.org/licenses/>.
+ * 
+ * Contributors:
+ *     wetransform GmbH <http://www.wetransform.to>
+ */
+
+package eu.esdihumboldt.hale.ui.common.crs;
```

Comment:
I don't think a UI bundle is the right place for these factories as we would like to use them in headless mode as well.

Probably requires a different approach to registering the factories. Maybe a fragment to the Geotools bundle that registers the factories via `META-INF/services`?

Reply:
Ah, didn't think of that, I'll look into it.